test/device/common.hpp

```diff
@@ -74,17 +74,17 @@ __global__ void fillKernel(DataType* data, uint32_t elementSize, uint32_t seed)
     {
         if constexpr(std::is_same_v<DataType, hipFloatComplex>)
         {
-            auto value  = (float(index / float(RAND_MAX) - 0.5) * 100) / elementSize;
+            auto value  = (float(index / float(RAND_MAX) - 0.5) * 10) / elementSize;
```

Comment:
Any specific reason to change the scale from 100 to 10?

Reply:
yes, will add a comment

Doing UnarySquare Operation for tensor(16F) may cause overflow.